frontend/kesaseteli/employer/package.json

```diff
@@ -13,8 +13,8 @@
     "test:debug-dom": "cross-env DEBUG_PRINT_LIMIT=1000000 yarn test",
     "test:staged": "yarn test --watchAll=false --findRelatedTests",
     "test:coverage": "yarn test:debug-dom --verbose --coverage",
-    "browser-test": "testcafe \"chrome --window-size='1249,720'\" testcafe.dotenv.js browser-tests/",
-    "browser-test:ci": "testcafe \"chrome:headless --disable-gpu --window-size='1249,720'\" --screenshots path=report,fullPage=true --video report --reporter spec,custom,html:report/index.html testcafe.dotenv.js browser-tests/"
+    "browser-test": "testcafe 'chrome --window-size=\"1249,720\" --ignore-certificate-errors-spki-list=\"8sg/cl7YabrOFqSqH+Bu0e+P27Av33gWgi8Lq28DW1I=,gJt+wt/T3afCRkxtMMSjXcl/99sgzWc2kk1c1PC9tG0=,zrQI2/1q8i2SRPmMZ1sMntIkG+lMW0legPFokDo3nrY=\"' testcafe.dotenv.js browser-tests/",
```

Comment:
review environment certificate hashes are added explicitly to make the browser test ignore specific certificate errors.